ui/apps/platform/src/Containers/Vulnerabilities/components/SummaryCardLayout.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { Alert, Gallery, GalleryItem, Skeleton } from '@patternfly/react-core';
+
+import { getAxiosErrorMessage } from 'utils/responseErrorUtils';
+
+const LoadingContext = React.createContext<{ isLoading: boolean }>({
+    isLoading: false,
+});
+
+export type SummaryCardProps<T> = {
+    data: T;
+    loadingText: string;
+    renderer: ({ data }: { data: NonNullable<T> }) => React.ReactNode;
```

Comment:
Only question is comment is related to breakthrough precedent for this pattern:
> Can be a render prop or a component.
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Components/TableStateTemplates/TbodyUnified.tsx#L18

Is **or a component** accurate?
* For table body rows with `map` method, it does not seem practical.
* The examples here include additional props and ofter use properties of `data` argument.

Therefore, maybe my question becomes pro and con of `({ data })` and `(data)` as argument?